### PR TITLE
[node] add a green circle as radio-range visualization of the selected node

### PR DESCRIPTION
**Pull request by Vinggui**
## Contribution description
When a node is selected, a green circle is drawn around it with the exact distance of the radio's range. It helps to place the node at a reachable distance from another device in the simulation.

## Code impact
No significant impact.

## Testing procedure
Same installation process as the original OTNS.

### CODE REVIEW

**Comment by simonlingoogle** [comment]
image

The feature looks great, thanks.

File: web/site/js/vis/Node.js [hunk 1]
```diff
@@ -105,41 +105,41 @@ export default class Node extends VObject {
         failedMask.scale.set(0.5, 0.5);
         failedMask.visible = false;
         this._root.addChild(failedMask);
-        this._failedMask = failedMask
+        this._failedMask = failedMask;
```
Comment (simonlingoogle) [inline web/site/js/vis/Node.js:108]: the style change seems unrelated to this PR. 

We currently don't have style enforcement on javascript. So, maybe better to enforce the style using a lint tool with a separate PR. 

Thoughts?

File: web/site/js/vis/Node.js [hunk 2]
```diff
@@ -287,15 +287,26 @@ export default class Node extends VObject {
             selbox.scale.set(selboxsize / NODE_SELECTION_SCALE, selboxsize / NODE_SELECTION_SCALE);
             selbox.anchor.set(0.5, 0.5);
             this.root.addChildAt(selbox, 0);
-            this._selbox = selbox
+            this._selbox = selbox;
+
+            const rangeCircleSize = this.radioRange;
+            let rangeCircle = new PIXI.Graphics();
+            rangeCircle.beginFill(0x20ff20, 0.2);
```
Comment (simonlingoogle) [inline web/site/js/vis/Node.js:294]: nit: maybe better choose a color of [Material Design](https://material.io/resources/color/#!/?view.left=0&view.right=0)

File: web/site/js/vis/Node.js [hunk 3]
```diff
@@ -287,15 +287,26 @@ export default class Node extends VObject {
...
+            rangeCircle.drawCircle(0, 0, rangeCircleSize);
```
Comment (simonlingoogle) [inline web/site/js/vis/Node.js:295]: nit: not sure if it would look better if the circle has a solid border line (e.g. using lineStyle).

File: web/site/js/vis/Node.js (hunk 1 above)
Comment (Vinggui) [reply web/site/js/vis/Node.js:108]: Yes, I agree. I got carried away with these semicolons, but you are right. I'm not going to make the PR for this style problem, but I really this it is unnecessary to leave lines without semicolons in JavaScript.

File: web/site/js/vis/Node.js (hunk 2 above)
Comment (Vinggui) [reply web/site/js/vis/Node.js:294]: You mean going to this Material Design link and picking a color and copying the number? That is fine, I agree to do it. However, Material Design does not seem to be installed for OTNS so far. All used colors were picked by numbers (#xxyyzz).

File: web/site/js/vis/Node.js (hunk 3 above)
Comment (Vinggui) [reply web/site/js/vis/Node.js:295]: Good idea. Done.

File: web/site/js/vis/Node.js (hunk 1 above)
Comment (simonlingoogle) [reply web/site/js/vis/Node.js]: > Yes, I agree. I got carried away with these semicolons, but you are right. I'm not going to make the PR for this style problem, but I really this it is unnecessary to leave lines without semicolons in JavaScript.

my javascript editor did this when formatting, not sure why though.

File: web/site/js/vis/Node.js (hunk 2 above)
Comment (simonlingoogle) [reply web/site/js/vis/Node.js]: yes, they were selected from material design color tools too.

File: web/site/js/vis/Node.js (hunk 3 above)
Comment (simonlingoogle) [reply web/site/js/vis/Node.js:296]: Have you got a picture of the final version?

File: web/site/js/vis/Node.js (hunk 1 above)
Comment (Vinggui) [reply web/site/js/vis/Node.js]: Oh, yeah. It seems fine, I'm not a JS specialist to criticize anything. I was just expressing an internal rage of JS crazy rules. I didn't mean anything.

File: web/site/js/vis/Node.js (hunk 2 above)
Comment (Vinggui) [reply web/site/js/vis/Node.js]: Thanks for explaining.

File: web/site/js/vis/Node.js (hunk 3 above)
Comment (Vinggui) [reply web/site/js/vis/Node.js:296]: image

Comment (simonlingoogle) [reply web/site/js/vis/Node.js:296]: Looks great! Thanks.

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Review by jwhui — APPROVED** [review]
